packages/cli/lib/utils.js

```diff
@@ -103,6 +104,32 @@ exports.validateClassName = function(name) {
   return util.format('Class name is invalid: %s', name);
 };
 
+exports.remindAboutNamingRules = function(name, log) {
```

Comment:
Is `checkNamingConvention` a better name?

Reply:
Not in my mind. I understand `checkNamingConvention` to report an error when the name does not match certain pattern(s). This function just prints a warning to explain the user what will happen.

Reply:
Maybe `reportNamingIssues` or `warnInvalidName`?

Reply:
`warnInputNaming`?

Reply:
I'll go with `logNamingIssues` for consistency with `logClassCreation`.